### PR TITLE
cookbook re-org: "find data" and create programmatic placeholder chapter

### PR DESCRIPTION
**Pull request by jules32**
Hi @asteiker and @betolink and @andypbarrett , here is the first cookbook rerog for "find data"; there is a placeholder chapter where we can incorporate an approach with `earthaccess`.

See our coworking notes for a refresher on what we were thinking.

### CODE REVIEW

File: _quarto.yml [hunk 1]
```diff
@@ -39,10 +39,12 @@ website:
         href: cheatsheet.qmd      
       - section: how-tos/index.qmd
         contents: 
-      #    - section: how-tos/find-data/index.md
-      #      contents:
-          - text: "Find Data"
-            href: how-tos/find-data/earthdata_search.md
+          - section: how-tos/find-data/index.md
```
Comment (asteiker) [inline _quarto.yml:42]: Looks good.

File: _quarto.yml [hunk 2]
```diff
@@ -39,10 +39,12 @@ website:
...
+            contents:
+              - text: "Earthdata Search"
+                href: how-tos/find-data/earthdata_search.md
+              - text: "Programmatic Search" 
+                href: how-tos/find-data/programmatic.md
```
Comment (asteiker) [inline _quarto.yml:47]: Turn this into .ipynb.

File: how-tos/find-data/index.md [hunk 3]
```diff
@@ -2,4 +2,4 @@
 title: Find Data
 ---
 
-More details coming soon.
\ No newline at end of file
+There are several ways to find data; here we'll show with EarthData Search and also programmatically.
```
Comment (asteiker) [inline how-tos/find-data/index.md:5]:
```suggestion
There are several ways to find data; here we'll show with Earthdata Search and also programmatically. 
```

**Review by asteiker — CHANGES_REQUESTED** [review]
Looks good!

File: _quarto.yml (hunk 2 above)
Comment (jules32) [reply _quarto.yml]: Hi @asteiker this is taken care of now, see our most recent commit in this PR

**Review by asteiker — APPROVED** [review]

**Pull request merged**